plugin/src/main/java/net/dzikoysk/funnyguilds/feature/war/WarSystem.java

```diff
@@ -79,41 +79,23 @@ public void attack(Player player, Guild guild) {
             conquer(attacker, guild, user);
         }
         else {
-            String messageForAttacker = WarUtils.getMessage(Message.ATTACKER, guild);
-            String messageForAttacked = WarUtils.getMessage(Message.ATTACKED, attacker);
-
-            for (User member : attacker.getMembers()) {
-                member.sendMessage(messageForAttacker);
-            }
-
-            for (User member : guild.getMembers()) {
-                member.sendMessage(messageForAttacked);
-            }
+            attacker.broadcast(WarUtils.getMessage(Message.ATTACKER, guild));
+            guild.broadcast(WarUtils.getMessage(Message.ATTACKED, attacker));
         }
     }
 
-    public void conquer(Guild conqueror, Guild loser, User attacker) {
+    public static void conquer(Guild conqueror, Guild loser, User attacker) {
```

Comment:
Tutaj nie powinno być statica. Metoda korzysta z FunnyGuilds.getInstance(). Nie jest to util metoda bez zależności. W przyszłości refactor takich statycznych metod będzie cięższy, niż takich z instancją, która może przetrzymywać potrzebne fieldy.